common/Cargo.toml

```diff
@@ -6,3 +6,4 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+borsh = "0.9"
```

Comment:
nice, NEAR also uses borsh. At first, Borsh, sounds like a scientist name, but there's really nothing special in comparison to serde.
Borsh stands for Binary Object Representation Serializer for Hashing.

Reply:
haha, sounds like a scientist indeed, or a car brand